server/controllers/visitation.controllers.js

```diff
@@ -71,6 +72,21 @@ const VisitationController = {
       })
       .catch((error) => next(error));
   },
+
+  cancelVisitation(req, res, next) {
+    const visitationInfo = req.locals;
+    const { user } = req;
+    cancelVisitation({ visitationInfo, userId: user._id })
+      .then(({ visitation, mailDetails, property }) => {
+        const contentTop = `The visitation to ${property.name} on ${visitation.visitDate}, has been cancelled by ${user.lastName} ${user.firstName}. Reason: ${visitation.rescheduleLog[0].reason}. Visit your dashboard for more information.`;
```

Comment:
Not sure if we should use the `LastName FirstName` naming convention. In the international market, it is always presented as `FirstName LastName`. Maybe we should have an helper function to generate someone's full name so that it will be easier to update irrespective of the format.

```suggestion
        const contentTop = `The visitation to ${property.name} on ${visitation.visitDate}, has been cancelled by ${user.firstName} ${user.lastName}. Reason: ${visitation.rescheduleLog[0].reason}. Visit your dashboard for more information.`;
```